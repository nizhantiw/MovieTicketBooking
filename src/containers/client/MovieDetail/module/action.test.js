import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme, { shallow, configure } from 'enzyme';
import renderer from 'react-test-renderer'
import actions from './actions'
configure({adapter: new Adapter()});

it("renders correctly", () => {
  const wrapper = shallow(
    <actions />
  );
  expect(wrapper).toMatchSnapshot();
});

it('renders a snapshot', () => {
  const tree = renderer.create(<actions/>).toJSON();
  expect(tree).toMatchSnapshot();
});

