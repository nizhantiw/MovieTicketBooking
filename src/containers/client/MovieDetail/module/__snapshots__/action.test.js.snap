// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `<actions />`;

exports[`renders correctly 1`] = `ShallowWrapper {}`;
